app/src/components/ActiveCallBanner/hooks.ts

```diff
@@ -0,0 +1,26 @@
+import { useNavigation } from '@react-navigation/native';
+import React from 'react';
+import { match } from 'ts-pattern';
+import {
+  useActiveCall,
+  useActiveCallRemoteParticipant,
+  useActiveCallDuration,
+} from '../../hooks/activeCall';
+import { type StackNavigationProp } from '../../screens/types';
+
+export const useConnectedActiveCallBanner = () => {
+  const navigation = useNavigation<StackNavigationProp<'App'>>();
+
+  const onPress = React.useCallback(() => {
+    navigation.navigate('Call');
+  }, [navigation]);
+
+  const activeCall = useActiveCall();
+  const title = useActiveCallRemoteParticipant(activeCall);
```

Comment:
```suggestion
  const title = getActiveCallRemoteParticipant(activeCall);
```

I think `get` would be a better prefix to use here since you are just trying to get a string.

Reply:
I agree, but React enforces a strict rule that hooks (and sub-hooks such as `useActiveCallRemoteParticipant`) need to start with `use` so they are clearly hooks. If I change the name to `get`, then the linter will error.